flake8_async/visitors/visitor4xx.py

```diff
@@ -0,0 +1,96 @@
+"""4XX error classes, which handle exception groups.
+
+ASYNC400 except-star-invalid-attribute checks for invalid attribute access on except*
+"""
+
+from __future__ import annotations
+
+import ast
+from typing import TYPE_CHECKING, Any
+
+from .flake8asyncvisitor import Flake8AsyncVisitor
+from .helpers import error_class
+
+if TYPE_CHECKING:
+    from collections.abc import Mapping
+
+EXCGROUP_ATTRS = (
+    # from ExceptionGroup
+    "message",
+    "exceptions",
+    "subgroup",
+    "split",
+    "derive",
+    # from BaseException
+    "args",
+    "with_traceback",
+    "add_notes",
+)
```

Comment:
Let's add a test that `set(dir(ExceptionGroup)).issubset(EXCGROUP_ATTRS)`, so that if a new method or attribute is added (e.g. by [PEP-785](https://peps.python.org/pep-0785/) 😅) we'll notice as soon as we test on that version of Python, rather than when users complain.

Reply:
oopie, that caught a typo. Turns out it's `add_note` and not `add_notes` ^^;